Volume/Views/DebriefArticleView.swift

```diff
@@ -0,0 +1,136 @@
+//
+//  DebriefArticleView.swift
+//  Volume
+//
+//  Created by Amy Chin Siu Huang on 12/2/21.
+//  Copyright © 2021 Cornell AppDev. All rights reserved.
+//
+
+import AppDevAnalytics
+import Combine
+import SwiftUI
+import SDWebImageSwiftUI
+
+struct DebriefArticleView: View {
+    @State private var cancellableShoutoutMutation: AnyCancellable?
```

Comment:
I think `cancellable` is what we do elsewhere. You only have 1 here anyway so I think being descriptive like this is overkill. Not a big deal though if you disagree